packages/webdriverio/src/commands/element/isStable.ts

```diff
@@ -0,0 +1,69 @@
+import { getBrowserObject } from '../../utils/index.js'
+
+/**
+ *
+ * Will return true when stable (in animation) or when unstable (not in animation)
+ *
+ * <example>
+    :index.html
+    <head>
+    <style>
+        div {
+        width: 200px;
+        height: 200px;
+        background-color: red;
+        animation: 3s 0s alternate slidein;
+        }
+        @keyframes slidein {
+        from {
+            margin-left: 100%;
+            width: 300%;
+        }
+
+        to {
+            margin-left: 0%;
+            width: 100%;
+        }
+        }
+    </style>
+    </head>
+    <body>
+    <div></div>
+    </body>
+
+    :isStable.js
+    it('should detect if an element is stable', async () => {
+        let element = await $('#has-animation');
+        console.log(await element.isStable()); // outputs: true
+
+        element = await $('#has-no-animation')
+        console.log(await element.isStable()); // outputs: false
+    });
+ * </example>
+ *
+ * @alias element.isStable
+ * @return {Boolean} true if element is stable, false if unstable
+ * @type state
+ *
+ */
+export async function isStable (this: WebdriverIO.Element) {
+    const browser = getBrowserObject(this)
+    return await browser.executeAsync((elem, done) => {
+        try {
+            const previousPosition = elem.getBoundingClientRect()
+            requestAnimationFrame(() => {
```

Comment:
Browsers will never call `window.requestAnimationFrame()` callback for background tab(performance optimisation) and drivers will throw meaningless `script timeout error` after `script timeout` time. So if user calls `isStable()` for element in inactive tab, it will be nice to inform him about it somehow in a friendly way.

You can easily reproduce it through test cases such as

```
browser.url("someurl")
$("some <a target="_blank"></a> element").click()
$("any element").isStable() // script timeout error here
```

You can check that the current window is backgroud by the following condition `document.visibilityState === 'hidden'`

P.S. It would also be nice to have an e2e test for such a case or comment on the code why we check the background tab or not.